src/components/playlist-wrapper/index.tsx

```diff
@@ -90,19 +77,11 @@ export const PlaylistWrapper = withText(translates)(
       },
       [onClose]
     );
-    const handleFocus = useCallback(() => {
-      setScrolling(true);
-    }, []);
-    
-    const handleBlur = useCallback(() => {
-      setScrolling(false);
-    }, []);
-    
-    const handleKeyDown = useCallback((event: KeyboardEvent) => {
-      if (event.keyCode === KeyMap.UP || event.keyCode === KeyMap.DOWN || event.keyCode === KeyMap.LEFT || event.keyCode === KeyMap.RIGHT) {
-        setScrolling(true);
-      }
-    }, []);
+
+    const handleFocus = useCallback(() => {}, []);
```

Comment:
if that's only `noop` methods, why keep them? Can we remove `handleFocus`, `handleBlur` and `handleKeyDown` methods?

Reply:
Done